bin/config-yargs.js

```diff
@@ -22,6 +22,12 @@ module.exports = function(yargs) {
 				describe: "Enviroment passed to the config, when it is a function",
 				group: CONFIG_GROUP
 			},
+			"version": {
+				type: "string",
+				describe: "Print the webpack version",
```

Comment:
I would write `describe` in a similar form as other entries so maybe just `webpack version`. I'm not sure if printing needs mention.